packages/core/src/components/forms/label.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+
+export interface ILabelProps extends React.HTMLProps<HTMLDivElement>, IProps {
+    /**
+     * Whether the label is non-interactive.
+     * Be sure to explicitly disable any child controls as well.
+     */
+    disabled?: boolean;
+
+    /** The helper text to show next to the label. */
+    helperText?: JSX.Element | string;
```

Comment:
Supports JSX.Element too, in case you want to **bold** or _italicize_ things.

Reply:
use `React.ReactNode` to support anything renderable.

Reply:
`React.ReactNode` also support arrays of elements though (and `ReactPortal` and other weird stuff). I'd rather explicitly spell out only the stuff we care about.

Reply:
but technically those values are supported! this is the crux of #1813